### PR TITLE
Show service class tags in order service wizard

### LINKED ISSUE #54: Order wizard should show service class tags
See https://github.com/openshift/origin-web-catalog/pull/50#issuecomment-290298152

cc @serenamarie125

### PR DESCRIPTION
**Pull request by spadgett**
screen shot 2017-03-30 at 12 58 45 pm

Fixes #54

cc @serenamarie125

### CODE REVIEW

File: src/styles/order-service.less [hunk 1]
```diff
@@ -21,24 +23,32 @@
   margin-top: 20px;
 
   .order-service-deails-top {
-    height: 70%;
-    .service-title {
-      display: flex;
-      align-items: center;
-      .icon {
+    .icon {
```
Comment (jwforres) [inline src/styles/order-service.less]: i noticed this earlier but didnt want to touch it, there is a typo in class name order-service-details-top

File: src/styles/order-service.less [hunk 2]
```diff
@@ -52,17 +62,13 @@
...
-      color: @color-pf-black;
-      background-color: @color-pf-white;
-      border-color: @color-pf-blue-300;
-    }
+  .order-service-description {
+    clear: both;
+    margin-top: 15px;
+  }
+  .tag {
+    margin-right: 5px;
```
Comment (jwforres) [inline src/styles/order-service.less]: should this be on a .tag + .tag selector as a margin-left so we dont have a hanging 5px margin on the last one

File: src/styles/order-service.less (hunk 1 above)
Comment (spadgett) [inline src/styles/order-service.less]: I can fix since I'm chaning it anyway

File: src/styles/order-service.less (hunk 2 above)
Comment (spadgett) [inline src/styles/order-service.less]: But if the tags wrap then you will have extra left margin on the second row

Comment (spadgett) [inline src/styles/order-service.less]: I could add negative margin to the block and then left margin to all tags

Comment (jwforres) [inline src/styles/order-service.less]: good point

Comment (jwforres) [inline src/styles/order-service.less]: um i would say leave it as is unless we find its causing an issue, lets not use negative margin unless we really need to

Comment (spadgett) [inline src/styles/order-service.less]: I have it working and it seems to work well, but I can take it back out. FWIW I think Bootstrap uses the same technique in its grid

**Comment by jwforres** [comment]
I'll let you decide

**Pull request merged**